src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int QUANTITY_INDEX = 1;
+    private static final int ACTION_INDEX = 0;
+    private static final String SUPPLY_ACTION = "supply";
+    private static final String BUY_ACTION = "buy";
+    private int supply = 0;
+    private int buy = 0;
+    private int result = 0;
+    private StringBuilder builder = new StringBuilder();
```

Comment:
builder is a bad name, doesn't tell us anything about its content
also, not sure they should be fields, we don't need to preserve them after writing to file